glue/main.py

```diff
@@ -45,17 +49,26 @@ def load_plugins(splash=None, require_qt_plugins=False):
     config = PluginConfig.load()
 
     n_plugins = len(list(iter_plugin_entry_points()))
+    if plugins_to_use is None:
+        if require_qt_plugins:
+            plugins_to_use = [*REQUIRED_PLUGINS, *REQUIRED_PLUGINS_QT]
+        else:
+            plugins_to_use = REQUIRED_PLUGINS
```

Comment:
I don't think ``plugins_to_use`` should be based on the ``REQUIRED_*`` variables - these are just used to raise exceptions below if they are not present. I think we actually need two different variables inside this function - ``plugins_to_load``(perhaps a better name than ``plugins_to_use``) and ``plugins_to_require``. I think the logic should be:

* If ``plugins_to_load`` is not defined by the user, then ``plugins_to_load`` defaults to all entry points (``iter_plugin_entry_points``) and ``plugins_to_require`` defaults to either ``REQUIRED_PLUGINS`` or the combination of ``REQUIRED_PLUGINS`` and ``REQUIRED_PLUGINS_QT``
* If ``plugins_to_load`` is defined by the user, then ``plugins_to_require`` should just be set to ``plugins_to_load`` because if the user is explicitly asking for them we should error if they can't be loaded